README.md

```diff
@@ -6,21 +7,27 @@ UI component for retrieving DataHub archived files.
 
 * Install Python 3
 * Create your virtual environment (venv)
-* Install Node 16.19.0
+* Install Node 16.19.0 (I recommend using nvm)
 * Install dependencies
 
 ```
+source /path/to/new/virtual/environment/bin/activate
 pip3 install -r requirements.txt [--upgrade]
 ```
 
 ### Front-end dependencies
 
 ```
-npm install
+npm install --save
```

Comment:
@cesarvh - I don't think `--save` is needed here. Please let me know if I'm wrong.  https://stackoverflow.com/questions/19578796/what-is-the-save-option-for-npm-install

Reply:
Ah, CSpace UI is still on npm 5.0, so I thought I was just used to doing it this way. Today I learned! Will remove